xml/System.Diagnostics/DiagnosticListener.xml

```diff
@@ -192,8 +192,8 @@
         <param name="name">To be added.</param>
         <param name="arg1">To be added.</param>
         <param name="arg2">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Override abstract method. Meant to check if the DiagnosticListener is enabled.</summary>
```

Comment:
```suggestion
        <summary>Checks if the DiagnosticListener is enabled.</summary>
```
But I'd consider removing this documenation for the same reason as `IsEnabled(String)`.